config/webpack.config.js

```diff
@@ -56,6 +57,7 @@ const commonConfig = ({ dev }) => {
         }
       : {}),
     devtool: false,
+    target: ['web', 'es4'],
```

Comment:
Why do we need the `es4`?

Reply:
Without it, Webpack will generate the `runtime` chunk using, at the very least, an arrow function, which is an ES6 feature. I figured it was good to set this as low as possible to maximize the chances old browsers at least get to display the outdated browser warning. I'm happy to remove it if you'd prefer.

Reply:
I think we can keep the web, but I'd remove the es4. It's way too old and PF is not supporting these browsers anyway. We won't lose any users over this.

Reply:
Ah, sorry. That's the default, so I'll just remove it.